maven/rules.bzl

```diff
@@ -349,7 +367,7 @@ def _deploy_maven_impl(ctx):
         runfiles = ctx.runfiles(files=files, symlinks = symlinks)
     )
 
-deploy_maven = rule(
+_deploy_maven = rule(
```

Comment:
This rule is privated and turned into a `py_binary` by a macro rather than setting the executable to a python script, which is not portable.